src/components/ReportTransaction.js

```diff
@@ -63,10 +88,24 @@ class ReportTransaction extends Component {
                                 styles.buttonSmall,
                                 styles.chatItemComposeSecondaryRowOffset,
                                 styles.mb3,
+                                {minWidth: '20%'},
```

Comment:
We shouldn't define inline styles: https://github.com/Expensify/Expensify.cash/blob/main/STYLING.md#inline-styles